servers/rendering_server.h

```diff
@@ -1402,7 +1402,7 @@ class RenderingServer : public Object {
 		INSTANCE_VOXEL_GI,
 		INSTANCE_LIGHTMAP,
 		INSTANCE_OCCLUDER,
-		INSTANCE_VISIBLITY_NOTIFIER,
+		INSTANCE_VISIBILITY_NOTIFIER,
```

Comment:
This is not correct, this breaks compatibility

Reply:
Compatibility with e.g. extensions that happen to use this identifier, I assume?

If the Godot policy is that typos in published identifiers are to be set in stone _forever_, I can roll this (and the other one below) back from this PR. Otherwise, I can also split those changes to a PR that could target whatever is the next major version where such a change is accepted.

Reply:
This change can't be made until 5.0 so please restore

Reply:
Alright 👍

Should I file a tracking issue to remind to fix this for the 5.0 cycle?

Reply:
This enum is exposed to scripting, so changing its name would break all scripts that use it.
That's a major compatibility break, which we don't allow between minor versions. If you want to rename it, both names should be available, with the old one wrapped in `#ifndef DISABLE_DEPRECATED`.

Reply:
I removed this (and the AJACENCY change below) from this PR, but added a TODO comment where the enum values are declared.